Source/ZoomNet/Resources/Meetings.cs

```diff
@@ -529,6 +529,43 @@ public Task<RegistrantInfo> AddRegistrantAsync(long meetingId, string email, str
 				.AsObject<RegistrantInfo>();
 		}
 
+		/// <summary>
+		/// Register up to 30 registrants at once for a meeting that requires registration.
+		/// </summary>
+		/// <param name="meetingId">The meeting ID.</param>
+		/// <param name="registrants">An array of registrants.</param>
+		/// <param name="autoApprove">Indicates if the registrant should be automatically approved.</param>
+		/// <param name="registrantsConfirmationEmail">Indicates if send confirmation Email to Registrants.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// An array of <see cref="BatchRegistrantInfo" />.
+		/// </returns>
+		public async Task<BatchRegistrantInfo[]> PerformBatchRegistrationAsync(long meetingId, BatchRegistrant[] registrants, bool autoApprove = false, bool registrantsConfirmationEmail = false, CancellationToken cancellationToken = default)
+		{
+			if (registrants.Any() == false || registrants.Count() > 30)
```

Comment:
You should make sure registrants is not null before checking the number of items in the enumeration.

Here's an example when I do something similar. I check that at least one item has been provided and no more than 5 (in your case the max value is 30, but otherwise the logic is the same): https://github.com/Jericho/ZoomNet/blob/develop/Source/ZoomNet/Resources/Chat.cs#L187-L188